Tests/MacroToolkitTests/DeclGroupTests.swift

```diff
@@ -160,4 +160,84 @@ final class DeclGroupTests: XCTestCase {
         XCTAssertEqual(testClass.accessLevel, .public)
         XCTAssertEqual(testClass.declarationContext, nil)
     }
+    
+    func testEnumRawTypeInferredValueString() throws {
+        let decl: DeclSyntax = """
+            enum TestEnum: String { case caseOne, caseTwo, caseThree = "case3" }
```

Comment:
Make at least one of these tests define the cases with multiple `case` statements to cover that situation as well.